drivers/wifi/nrf700x/zephyr/src/zephyr_wpa_supp_if.c

```diff
@@ -1016,3 +1016,89 @@ void wifi_nrf_wpa_supp_event_proc_get_if(void *if_priv,
 
 	k_sem_give(&wait_for_event_sem);
 }
+
+void wifi_nrf_wpa_supp_event_mgmt_tx_status(void *if_priv,
+		struct nrf_wifi_umac_event_mlme *mlme_event,
+		unsigned int event_len)
+{
+	struct wifi_nrf_vif_ctx_zep *vif_ctx_zep = NULL;
+
```

Comment:
Good to have a defensive check here to see that if_priv and mlme_event are not NULL.